src/DataBoxEdge/DataBoxEdge.Test/ScenarioTests/Common.ps1

```diff
@@ -48,7 +48,8 @@ Returns standard EncryptionKey
 #>
 function Get-EncryptionKey
 {
-	$encryptionKey = ConvertTo-SecureString -String ""
```

Comment:
correct me if I was wrong, I think this line will fail immediately, and this test case is in live only mode so for your new changes, there are no way for CI to verify them.

Reply:
Yes, since this involves adding a secret in the code base, I need to migrate to azure keyvault. Currently, I am testing everythin locally by adding the encryption keys and running tests.

Removing the encryption key before making a commit